### DISCUSSION TITLE
下载不了游戏怎么办

### DISCUSSION THREAD

**Discussion by 1249qwe** [Meloong-Git/PCL#1507] [提问] [answered]
如题，下载的时候会卡在原版json

**Comment by Pigeon0v0** [comment]
给个截图

**Reply by 1249qwe** [reply]
![正在上传 032618304924_0IMG_20230326_181812.jpg…]()

**Reply by WForst-Breeze** [reply]
没传上来，您拖拽图片后稍等一会再发送

**Reply by 1249qwe** [reply]
![032618304924_0IMG_20230326_181812.jpg](https://user-images.githubusercontent.com/128267286/227770653-c8a25609-7fc5-4b6f-9325-37ce6faebd07.jpg)

**Reply by ZmdNotFound** [reply]
试着清理缓存，或者使用VPN？

请合理运用你的截图键，谢谢

**Comment by LTCatt** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/40348278/227786119-5a89b392-4ddb-41ff-ae35-0acc22c124bb.png)
